sdk/src/lib.rs

```diff
@@ -41,8 +41,10 @@ pub use val::Val;
 pub use vec::Vec;
 
 #[inline(always)]
-pub fn require(b: bool) {
-    b.or_abort();
+pub const fn require(b: bool) {
+    if !b {
+        panic!();
```

Comment:
It seems like there's no value in calling `trap` directly via `or_abort`. Calling it directly can't be done from const functions because the wasm unreachable function is not const. Calling panic seems to do the same thing.